pyproject.toml

```
@@ -22,7 +22,7 @@ dependencies = [
     "pyaudio ~= 0.2.13",
     "pydub ~= 0.25.1",
     "rich ~= 13.4",
-    "shazamio >= 0.4.0.1,< 0.5.2.0",
+    "shazamio >= 0.6.0",
```

Nitpick: compatible-release constraint instead:

```suggestion
    "shazamio ~= 0.6.0",
```